.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,113 @@
+name: Testsuite
+
+on: [push]
+
+jobs:
+  flake8:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version:  [3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
+        echo "::set-output name=dir::$(pip cache dir)"
+    - name: pip cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.pip-cache.outputs.dir }}
+        key: py${{ matrix.python-version }}-${{ matrix.os }}-pip
+
+    - name: Syntax errors check
+      run: |
+        python -m pip install flake8
+        flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+    - name: Style warnings
+      run: |    
+        # exit-zero treats all errors as warnings.
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=79 --statistics        
+
+  build:
+    needs: flake8
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        #  ** Update flake8 literals if change os or python-versions
+        os: [ubuntu-latest, windows-latest] # , macos-latest]
```

Comment:
I did temporarily during testing.  I had planned to put it back for the 'merge' workflow only, since really it is basically linux, so little chance of pydicom code causing an error in mac and not in linux.

I was thinking of PyPy too (they have that as a stock OS) but I'm not sure what we can set up with that -- probably not gdcm but maybe the others can run.  I'm not very familiar with setting it up and running it though.  And I kind of feel that with Python typing, maybe some other speed-up solutions will be appearing.  In any case, will probably leave for its own PR.